api/api/routers/openai_proxy/_openai_proxy_handler.py

```diff
@@ -468,15 +468,18 @@ def _feedback_generator(run_id: str) -> str:
     @classmethod
     async def missing_model_error(cls, model: str | None, prefix: str = ""):
         _check_lineup = f"Check the lineup 👉 {WORKFLOWAI_APP_URL}/models ({MODEL_COUNT} models)"
```

Comment:
```suggestion
        _check_lineup = f"Check the lineup 👉 {WORKFLOWAI_APP_URL} ({MODEL_COUNT} models)"
```

(Removed trailing slash compared to your suggestion)